sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -159,6 +159,12 @@ namespace Azure { namespace Core { namespace Http {
      */
     AZ_CORE_DLLEXPORT const static HttpMethod Patch;
 
+    /**
+     * @brief The representation of a `OPTIONS` HTTP method based on [RFC 2616]
+     * (https://datatracker.ietf.org/doc/html/rfc5789).
```

Comment:
We reference an old/obsoleted RFC above (2616) but link to RFC 5789 here. What's the right and current RFC to reference?